gallery/how-to/managing-profile/managing-account.md

```diff
@@ -6,82 +6,94 @@ title:  PowerShell Gallery Account Settings
+- Account management notifications from PowerShell Gallery administrators
+- Notifications about the tests run by the PowerShell Gallery for submissions you have made
 
-## Change Login Account
+## Change your login account
 
-Changing the login account requires you to be logged in with the current account to access Account Settings, but then logging in with the new account you wish to switch to. The sequence of steps are shown below.
+To change the login account, you must be signed in with the current account. Use the following
+steps to complete the change.
 
 ![Login Account settings](../../Images/PSGallery_LoginAccountSettings.png)
 
-1. When you click on "Change Account", a pop-up will be shown to explain that changing the login account applies to all uses of that account in the PowerShell Gallery. Review the information, then click "OK" to continue. 
+1. Click on **Change Account**. A pop-up window explains that changing the login account applies to
+   all uses of that account in the PowerShell Gallery. Review the information, then click **OK** to
+   continue.
 
-![Login Account settings](../../Images/PSGallery_LoginAccountChange-1.png)
+   ![Login Account settings](../../Images/PSGallery_LoginAccountChange-1.png)
 
-2. You will then be prompted to log in, and must enter the **new account** you wish to use as your login account, as shown below.
+2. You are then prompted to sign in using the _new account_.
 
-![Login Account settings](../../Images/PSGallery_LoginAccountChange-2.png)
+   ![Login Account settings](../../Images/PSGallery_LoginAccountChange-2.png)
 
-3. You will see another prompt that you are signed in with the current account. Select "Sign out and sign in with a different account." 
+3. When you click **Next**, you see a message that you are signed in using the current account.
+   Click **Sign out and sign in with a different account**.
 
-![Login Account settings](../../Images/PSGallery_LoginAccountChange-3.png)
+   ![Login Account settings](../../Images/PSGallery_LoginAccountChange-3.png)
 
-4. You will then be prompted for the password of the new account, to complete the process. Once you enter the password, you will be returned to the Account Settings page, which will show that the login account has been updated.
+4. Enter the password of the new account. After entering the password, you are returned to the
+   Account Settings page showing you that the login account has been updated.
 
 
 ## Enable Two-Factor Authentication (2FA)
 
-Enabling two-factor authentication is a recommended security measure for all users who publish manually to the PowerShell Gallery. 
-This change will require that your login account has at least two forms of authentication registered for it. One will be the password, the other may be 
+Two-factor authentication is recommended for all users who publish manually to the PowerShell
+Gallery. To enable two-factor authentication, login account must have at least two forms of
```

Comment:
I think "the" is missing in:
To enable two-factor authentication, the login account...

Reply:
fixed